src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -249,14 +256,14 @@ protected virtual void PostLogFile(IAnalysisContext globalContext)
             }
         }
 
-        internal static async Task PostLogFile(string postUri, string outputFilePath, IFileSystem fileSystem, HttpClient httpClient)
+        internal static async Task<bool> PostLogFile(string postUri, string outputFilePath, IFileSystem fileSystem, HttpClient httpClient)
         {
             if (string.IsNullOrWhiteSpace(postUri))
             {
-                return;
+                return false;
             }
 
-            await SarifLog.Post(new Uri(postUri), outputFilePath, fileSystem, httpClient);
+            return await SarifLog.Post(new Uri(postUri), outputFilePath, fileSystem, httpClient);
```

Comment:
I think this is correct right now, but it's a maintenance hazard.  If someone reasonably adds another reason to return false below, this line will continue reporting empty log files that might not be empty.

Can you move the Console output down to where the post-or-not decision is being made?

Reply:
Oh, if you do this, I think you can remove the return code, which simplifies things nicely.

Reply:
Console output down to where the post-or-not decision is being made ---- sounds good, fixed